deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -72,13 +72,16 @@ data:
         @type protobuf
       </parse>
     </source>
-    <match prometheus.metrics>
+    <match prometheus.metrics.**>
       @type datapoint
-      tag "prometheus.datapoint"
+      tag prometheus.datapoint
     </match>
     <filter prometheus.datapoint>
       @type carbon_v2
-    </filter>    
+    </filter>
+    # <filter prometheus.datapoint>
```

Comment:
I think we should remove the comment

Reply:
It's for easier debugging. I'm ok to remove it